packages/connect-node-test/src/crosstest/unresolvable_host.spec.ts

```diff
@@ -28,14 +30,31 @@ import { TestService } from "../gen/grpc/testing/test_connect.js";
 import { PayloadType } from "../gen/grpc/testing/messages_pb.js";
 
 describe("unresolvable_host", function () {
-  const optionsHttp2 = {
+  const baseOptions = {
     baseUrl: "https://unresolvable-host.some.domain",
+  };
+  const optionsHttp2 = {
+    ...baseOptions,
     httpVersion: "2" as const,
   };
   const optionsHttp1 = {
-    baseUrl: "https://unresolvable-host.some.domain",
+    ...baseOptions,
     httpVersion: "1.1" as const,
   };
+  const unresolvableRouter = ({ service }: ConnectRouter) => {
+    const unavailableRoute = () => {
+      throw new ConnectError(
+        "getaddrinfo ENOTFOUND unresolvable-host.some.domain",
+        Code.Unavailable
```

Comment:
This works 👍 

But by definition, the router transport only serves the routes you give to it. Any other route will hit raise this error: https://github.com/bufbuild/connect-es/blob/fc9cd63226289cc792c0e108c1aeefd802216510/packages/connect/src/protocol/universal-handler-client.ts#L36-L39

I can't think of a meaningful way to run the router transport with this test case, I think we can simply omit it here.

Could you instead add a test in packages/connect/src/router-transport.spec.ts to cover the behavior for calling an RPC with the router transport that isn't registered?

Reply:
> I think we can simply omit it here.

sure: removed it in 7042dd57ad41a7d6191093b2417b8c57264d9208

Reply:
> Could you instead add a test in packages/connect/src/router-transport.spec.ts to cover the behavior for calling an RPC with the router transport that isn't registered?

sure: 3bdbd362a8fbb959fa927245374d9a1f250b25a2